README.md

```diff
@@ -1420,8 +1420,6 @@ Background sync is not just hard, it is stupid. Here are my technical and philos
 
 ## Known Issues
 
-* It probably doesn't work on Windows.
```

Comment:
Yeah unfortunately I personally don't have Windows machines nor would I be able to verify that it works properly on Windows so I am hesitant about removing this statement.

Reply:
I've added back a modified statement: that it's unsupported but works somewhat.